client/src/app.jsx

```diff
@@ -1,57 +1,82 @@
+import ProductCategory from './components/productCategory';
+import { getAllStyles, getInfo } from './lib/routes';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
+const App = () => {
+  const [styles, setStyles] = useState({});
+  const [productId, setId] = useState(1);
+  // const [products, setProducts] = useState([]);
+  const [product, setProduct] = useState({});
 
-    };
-    // this.getProducts = this.getProducts.bind(this);
-  }
-  // getProducts((error, data) => {
-  //   if (error) {
-  //     throw(error)
-  //   } else {
-  //     console.log(data.results);
-  //   }
-  // })
+  useEffect(() => {
+    // console.log('this time!');
+    getAllStyles((error, response) => {
+      // console.log('data within get all styles', response.data);
+      if (error) {
+        return 'Could not get styles';
+      }
+      return setStyles(response);
+    });
+    // listProducts((error, response) => {
+    //   if (error) {
+    //     return 'Could not get products';
+    //   }
+    //   return setProducts(response);
+    // });
+    getInfo((error, response) => {
+      if (error) {
+        return 'Could not get products';
+      }
+      // console.log('id in app: ', response.id);
+      setId(response.id);
+      return setProduct(response);
+    });
+  }, []);
 
-  render() {
-    return (
+  return (
+    <Container>
+      <TitleImage />
       <Container>
-        <br />
-        <TitleImage />
-        <Container>
-          <Row>
-            <MainImage />
-            Stars will go here
-            <br />
-            Category will go here
-            <br />
-            Product Name will go here
-            <br />
-            Price will go here
-            <br />
-            Style and Selected Style will go here
-            <br />
-            Style Thumbnails will go here
-            <br />
-            Size and Quantity Dropdowns will go here
-            <br />
-            Add to Cart and Favorite Buttons will go here
-            <br />
-          </Row>
-        </Container>
-        <Container>
-          Product Description will go here
-        </Container>
+        <Row>
+          <Col xs={12} sm={12} md={8} lg={8}>
+            <MainImage styles={styles} />
+          </Col>
+          <Col xs={12} sm={12} md={4} lg={4}>
+            <Row>
+              <span role="img" aria-label="stars">⭐️⭐️⭐️⭐️⭐️</span>
+              Stars will go here
+            </Row>
+            <Row><ProductCategory product={product} /></Row>
+            <Row><ProductTitle product={product} /></Row>
+            <Row>Price will go here</Row>
+            <Row>Style and Selected Style will go here</Row>
+            <Row>Style Thumbnails will go here</Row>
+            <Row>Size and Quantity Dropdowns will go here</Row>
+            <Row>Add to Cart and Favorite Buttons will go here</Row>
+          </Col>
+        </Row>
       </Container>
-    );
-  }
-}
+      <Container>
```

Comment:
would help to modularize your code by moving this container and its contents into its own component

Reply:
Great idea!